Testing GitHub

JosephGreene opened the pull request:
I added a comment saying 'Testing Github' into Spandex Stack so I ensure
I opened this branch correctly. Please ignore this Derin, I'm just making sure I'm not fucking up master just yet :smile:

JosephGreene committed:
Testing GitHub

I added a comment saying 'Testing Github' into Spandex Stack so I ensure
I opened this branch correctly

derinsevenler commented:
That's fine! As you keep making commits to the Joe_Spandex_Single_and_Flicker branch they will show up here, leave a comment when you think the changes are ready to go!

JosephGreene committed:
Implimented Spandex Single V1

I also commented spandex stack so someone could use it as a tutuorial
and better understand the code behind it. Spandex_Single_Experimental is
my prototype testbench.

JosephGreene committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Joe_Spandex_Single_and_Flicker

# Conflicts:
#	Particle detection plugins/Spandex_Stack.java

JosephGreene committed:
experimental file for prototyping spandex single

JosephGreene committed:
Updated code

V1 now can handle the 0 particle condition.

Experimental now has a parameter where we can set the maximum threshold
limit, so we can filter noise.

JosephGreene committed:
Both V1 and Experimental now use 2d guassian blur

Program is significantly faster

JosephGreene committed:
V2 senses 'dark' particles

Boolean parameter which allows you to search for EITHER NP bighter or
darker than the background. V3 will do both.

JosephGreene committed:
V3 detects both bright and dark particles

However, taking proper actions if there are no particles is a bit
trickier, so experimental currently has a fix for that. The only reason
I didn't incorporate it into v3 is I get a weird ROIwindow error I'm
working out.

JosephGreene committed:
Spandex Single Complete!!!!!!

Turns out I just copied a piece of code twice I shouldn't have. Spandex
Single should work with no errors now.

JosephGreene committed:
Created a new SPANDEX

Spandex time analyzes image stacks were it is single images seperated by
some time constant. This is for use in kon and koff experiments.

JosephGreene committed:
Spandex Time functional on all images

Fixed Spandex time so it analyzes any image stack, and created a new
plug in called Image_Data, which reads in an image and returns all
details on it (Width, Height, Channels, Slice Number, Frame Number)

JosephGreene committed:
Creating Spandex Crop

Takes an image with multiple spots and splits it up, analyzing each spot
independantly
